docs/source/sawtooth_cli/admin_genesis.rst

```diff
@@ -0,0 +1,52 @@
+Admin Genesis CLI Tool
+======================
+
+Overview
+--------
+
+The genesis CLI tool produces a file for use during initialization of a
+Validator.  A network requires an initial block (known as the genesis block),
+whose signature will determine the block chain id.  This initial block is
+produced from a list of batches, which will be applied at genesis time.  The
+input to the command is a set of zero or more files containing serialized
+``BatchList`` protobuf messages. The output is a file containing a serialized
+``GenesisData`` protobuf message.  This file, when placed at
+``<SAWTOOTH_HOME>/data/genesis.batch``, will trigger the genesis process.
```

Comment:
It is SAWTOOTH_HOME/data/genesis.batch only if SAWTOOTH_HOME environment variable is set, otherwise it defaults to /var/lib/sawtooth/ (or it will, when we get that far).